test/utils/BatchLockupBuilder.sol

```diff
@@ -0,0 +1,232 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.8.22;
+
+import { LockupDynamic, LockupLinear, LockupTranched } from "@sablier/v2-core/src/types/DataTypes.sol";
+
+import { BatchLockup } from "../../src/types/DataTypes.sol";
+
+library BatchLockupBuilder {
+    /// @notice Generates an array containing `batchSize` copies of `batchSingle`.
+    function fillBatchLockup(
```

Comment:
Adding  `Lockup` is redundant in these functions. That this is about `BatchLockup` is implied by the name of the library (`BatchLockupBuilder`).